src/Message/Response.php

```diff
@@ -346,6 +346,19 @@ public function getSubscriptionStatus()
     }
 
     /**
+     * Get the billing state of the subscription.
+     *
+     * @return string|null
+     */
+    public function getSubscriptionBillingState()
+    {
+        if (isset($this->data->autobill)) {
```

Comment:
you can just do $this->getSubscription() instead of $this->data->autobill. See getSubscriptionId above